src/megaapi_impl.cpp

```diff
@@ -11388,13 +11386,12 @@ char *MegaApiImpl::getCRC(const char *filePath)
     string localpath;
     fsAccess->path2local(&path, &localpath);
 
-    FileAccess *fa = fsAccess->newfileaccess();
+    auto fa = fsAccess->newfileaccess();
     if(!fa->fopen(&localpath, true, false))
         return NULL;
 
     FileFingerprint fp;
-    fp.genfingerprint(fa);
-    delete fa;
```

Comment:
replace this with fa.reset()?

Reply:
nothing significant occurs before the end of the function here, so I think it's ok

Reply:
agreed